### DISCUSSION TITLE
The xiao rp2040 is Expensive

### DISCUSSION THREAD

**Discussion by sorrowfullys** [ravinrabbid/DonCon2040#8] [Q&A]
donconio can use rp 2040 zero 
but i don't know how to use it
![image](https://github.com/user-attachments/assets/1c52b6ed-0c6f-4836-bf09-e42bc9171bd4)
![image](https://github.com/user-attachments/assets/2bf54cb1-08ff-4571-ae08-677d2a2f32f9)
do you know the Needle correspondence?

**Comment by ravinrabbid** [comment]
The picture of the XIAO RP2040 you posted looks a bit off. You should probably refer to this one here https://wiki.seeedstudio.com/XIAO-RP2040/.

If I'm not mistaken, the Pxx pins on the XIAO should correspond to the GPxx pins on the WaveShare. You can also configure the pins in `GlobalConfiguration.h`, just make sure that they support the needed function (i2c, spi, ...) in this case.
